tests/tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function dashboard_admin_login {
+    touch $COOKIES
+    $CURL_BIN $LOGIN_URL > /dev/null
+    DJANGO_TOKEN="csrfmiddlewaretoken=$(grep csrftoken $COOKIES | \
+    sed 's/^.*csrftoken[[:blank:]]*//')"
+    $CURL_BIN -d "$DJANGO_TOKEN;username=$USERNAME;password=$PASSWORD" \
+    -X POST $LOGIN_URL
```

Comment:
increase the indentation to make sure is clear it's a line continuation, eg:

```
$CURL_BIN -d "$DJANGO_TOKEN;username=$USERNAME;password=$PASSWORD" \
          -X POST $LOGIN_URL
```

Reply:
this is great, but you should check the response in some way, as you're doing in the other tests, otherwise it looks like (at least to me) that it won't help much, what do you think?